service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskDefinition.java

```diff
@@ -1,11 +1,14 @@
 package uk.nhs.adaptors.gp2gp.gpc;
 
+import lombok.Data;
```

Comment:
checkstyle is failing the build that this import is no longer used. Please remove